src/Pages/JoinRoom/JoinRoom.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState  } from 'react';
+import { useNavigate } from "react-router-dom";
+
+import TextInput from '../../Components/TextInput/TextInput';
+
+import './JoinRoom.scss';
+
+function JoinRoomPage() {
+  const [roomId, setRoomId] = useState('');
+  const navigate = useNavigate();
+
+  const next = () => {
+    navigate('/join-room/:roomId')
```

Comment:
```suggestion
    navigate(`/join-room/${roomId}`)
```
Yönlendirmede kullanılan bağlantı static değil dinamik olmalı ki kullanıcı girdiği oda için olan isim giriş sayfasına yönlendirilebilsin